Classes/ViewHelpers/Widget/LinkViewHelper.php

```diff
@@ -67,13 +67,10 @@ public function initializeArguments()
      */
     public function render($pluginName, $extensionName, $action = null, $arguments = array(), $section = '', $format = '', $ajax = true)
     {
-        if ($ajax === true) {
-            $uri = $this->getAjaxUri();
-        } else {
-            $uri = $this->getWidgetUri();
-        }
+        $uri = $ajax ? $this->getAjaxUri() : $this->getWidgetUri();
```

Comment:
When not taking into consideration that Fluid correctly casts this variable to a boolean, then explicitly asking for a boolean true is different than assuming true for all other possible values PHP casts to be true ('1', 1, array('asd'), new \stdClass, …)